sdk/daml-lf/language/src/main/scala/com/digitalasset/daml/lf/language/UtilityDamlPrimPackages.scala

```diff
@@ -0,0 +1,1286 @@
+// Copyright (c) 2024 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.digitalasset.daml.lf.language
+
+import com.digitalasset.daml.lf.data.Ref._
+
+object UtilityDamlPrimPackages {
+  def check(pkgId: PackageId) = prims.contains(pkgId);
+
+  val prims: Map[String, (String, String)] =
+    Map(
+      (
+        "3697a9dababb68797d839216d0c70aec808756a39bdd95ee68a74309ac82eea0",
+        ("2.1", "3.0.0.20240212.12749.0"),
+      ),
+      (
+        "81a75fc7d4ddd5d6ed376018e4ad3ef7ab680d881a9542eb5dd5e8962fec283c",
+        ("2.dev", "3.0.0.20240212.12749.0"),
```

Comment:
We do not need to consider 2.dev.  
We do not provide any compatibility garantee of 2.dev package between release/snapshot (one can use dev pakcage only with the same snapshot/realease it produced it).

Reply:
makes sense from CN’s pov. We only need continuity across the stable versions not the dev versions.

Reply:
This should just be a list of all daml prim packages that should be marked as utility packages but aren't. I don't want to overcomplicate this with additional qualifiers - it doesn't cost us anything to be complete about the list.

Another thing I don't like about leaving out 2.dev packages is that it creates a strange halfway situation where we can rely on 2.dev packages after the August 23rd release, but not the ones before.

Reply:
I am concerned about making the code clearer.
I found that very confusing we specify 2.dev package when you just must **never ever** rely on old 2.dev packages. 
2.dev package produces by a given compiler can be use only by the runtime form the same SDK.